services/github/src/constants.js

```diff
@@ -11,6 +11,24 @@ const TASK_STATE_TO_CHECK_RUN_STATE = {
   completed: CHECK_RUN_STATES.COMPLETED,
 };
 
+const GITHUB_TASKS_FOR = {
+  PUSH: 'github-push',
+  RELEASE: 'github-release',
+  PULL_REQUEST: 'github-pull-request',
+  PULL_REQUEST_UNTRUSTED: 'github-pull-request-untrusted',
+};
+
+// https://docs.github.com/developers/webhooks-and-events/webhooks/webhook-events-and-payloads?actionType=released#release
+const GITHUB_RELEASE_ACTION = {
```

Comment:
Nice, definitely can be a follow-up, but if we're adding this here it would be good to also add `GITHUB_PULL_REQUEST_ACTION`.

Reply:
there are too many of those :) maybe we don't need it atm